fe/src/components/BodyParamsForm/index.jsx

```diff
@@ -88,8 +87,12 @@ export default function BodyParamsForm() {
         if (tdee) {
           localStorage.setItem('tdee', tdee)
         }
-
-        navigate('/bodyfat')
+        const is_first_time = res?.data?.is_first_time
+        if (is_first_time) {
+          navigate('/bodyfat')
+        } else {
+          navigate('/bodyfatv2')
+        }
```

Comment:
Nếu comment ở trên là đúng thì không cần cái này nữa, trong màn `/bodyfat` check kiểu:
```suggestion
if (is_first_time) {
  <Button />
  <Navbar />
} else {
  <Navbar />
}
```